### PR TITLE
Feat/implement design + link to data

### LINKED ISSUE #17: Implement design

### PR DESCRIPTION
**Pull request by ThomasCaud**
Resolve #17 #18

### CODE REVIEW

File: src/Components/Content/Content.js [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+        "highScoreDay": {
+          "date": data.highScoreDay,
+          "runtime": this.getDisplayedTime(data.highScoreDay.runtime)
+        }
+      }
+    }
+  }
+
+  // for testing purpose
+  /*componentDidMount () {
```
Comment (fberrez) [inline src/Components/Content/Content.js]: delete ?

File: src/Processor/Processor.js [hunk 2]
```diff
@@ -78,7 +79,8 @@ class Processor {
               dates: processedResults[3],
               highScoreDay: processedResults[4],
             };
-            console.log(`resume: ${JSON.stringify(resume, undefined, "  ")}`);
+            return resume;
+            // console.log(`resume: ${JSON.stringify(resume, undefined, "  ")}`);
```
Comment (fberrez) [inline src/Processor/Processor.js]: delete

File: src/Components/Content/Content.js (hunk 1 above)
Comment (ThomasCaud) [reply src/Components/Content/Content.js]: Done

File: src/Processor/Processor.js (hunk 2 above)
Comment (ThomasCaud) [reply src/Processor/Processor.js]: Done

**Review by fberrez — APPROVED** [review]

**Pull request merged**